### DISCUSSION TITLE
Commit activity is not displayed

### DISCUSSION THREAD

**Discussion by ArtemKostko1** [community/community#66974] [GitHub Education] [answered]
### Hi everyone,

I created a repository and after that I do not display the activity of the commits in the contributions section of the profile. The last activity displayed is the creation of this repository. I can’t figure out how to make my activity appear in this section.

**Comment by hafiz-muhammad** [comment, marked as answer]
Here's a GitHub documentation with common reasons why your contributions are not showing in your graph: [Why are my contributions not showing up on my profile?](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile)
